src/init/starship.ion

```diff
@@ -4,7 +4,7 @@ fn PROMPT
     let STARSHIP_ION_STATUS = $?
     # Save the ION_CMD_DURATION before running any other commands, but after the status since the
     # duration is not updated during variable assignment
-    let STARSHIP_ION_CMD_DURATION = $ION_CMD_DURATION
```

Comment:
Do you think it would be worth supporting both, for the time being, to avoid breaking it for folks using an older build of Ion?

We do that for fish right now:
https://github.com/starship/starship/blob/master/src/init/starship.fish#L9-L10

Reply:
Didn't find the correct command first time around. Now it falls back to the old variable.